modules/sdk-core/src/bitgo/staking/stakingWallet.ts

```diff
@@ -90,7 +90,7 @@ export class StakingWallet implements IStakingWallet {
    * @param transaction - staking transaction to build
    */
   async build(transaction: StakingTransaction): Promise<StakingPrebuildTransactionResult> {
-    if (this.wallet.baseCoin.supportsTss()) {
+    if (this.wallet.baseCoin.supportsTss() && this.wallet.baseCoin.getFamily() !== 'eth') {
```

Comment:
Can we document not supporting eth family in the jsdocs?

Reply:
staking team is aware of this change. This change is requires to keep eth as is today while we add support tss for eth. Denny Becker is going to put it on their road map to change this.